explorer.go

```diff
@@ -92,7 +100,30 @@ func init() {
 
 	if apiOnly {
 		log.Println("Running in api-only mode")
-	}
+    }
+    
+    // Load the last saved cache
+    file := filepath.Join( os.TempDir(), cacheFileName)
+    content, err := ioutil.ReadFile(file)
```

Comment:
Log the error (with filename) and a message whether loading cache from disk or not